lms/templates/courseware/course_updates.html

```diff
@@ -1,5 +1,6 @@
 <%! from django.utils.translation import ugettext as _ %>
-<section>
+<section aria-labelledby="course-updates-heading">
+    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
```

Comment:
Course updates appears here: https://clrux-ac-520.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/info (each update is like a post in a blog).